core/src/state.rs

```diff
@@ -97,6 +97,25 @@ pub enum State {
     Live(LiveState),
 }
 
+/// Options for [`to_ext`]
+///
+/// Check if runtime is compiled with try-runtime option or not
+#[derive(PartialEq, PartialOrd)]
+pub enum TryRuntimeFeatureCheck {
+    Check,
+    Skip,
+}
+/// Options for [`to_ext`]
+///
+/// Enable of disable check wether new runtime spec version is not greater
+/// than the on-chain runtime spec version in [`to_ext`]
+
+#[derive(PartialEq, PartialOrd)]
+pub enum SpecVersionCheck {
+    Check,
+    Skip,
+}
```

Comment:
Needs to be wrapped at col 100
```suggestion
/// Whether to check if the new runtime `spec_version` is greater than the previous runtime `spec_version`
#[derive(PartialEq, PartialOrd)]
pub enum SpecVersionCheck {
    /// Check that the new runtime `spec_version` is greater than the previous runtime `spec_version`
    Check,
    /// Don't check that the new runtime `spec_version` is greater than the previous runtime `spec_version`
    Skip,
}
```